### ISSUE TITLE
The "cert-manager hybrid mode" configuration difficult to get working

### ISSUE THREAD

**Issue by mflendrich** [Kong/charts#821]
@mheap reported that the quick start guide for hybrid mode does not work if you use cert-manager with ACME.

## Acceptance critieria
- [ ] As a user trying to deploy Kong hybrid mode from the Helm chart, assuming that I want to use ACME to obtain CP-DP and Admin API certificates, I want clear instructions (in the form of a quick start guide) to make this work.

**Comment by spjohn85** [comment]
This is true, I am trying to use ACME for hybrid deployment with PKI, the chart values given does not seem to fulfil the successful deployments. 

I have got it to working to provision the certs and everything (may be I can help document this once I tackle this last one), I am faced with a [CA_CERT](https://github.com/cert-manager/cert-manager/issues/2111) not available to use when using ACME. 

This is required for KONG_CLUSTER_CA_CERT to validate the connections for mTLS. 

![image](https://github.com/Kong/charts/assets/36644947/c225a22b-3d91-45ec-9a0e-c2a05e54eed9)

@mflendrich Anyone successful in this part ?
